app/routers/speaker_diarization.py

```
@@ -0,0 +1,142 @@
+                    timestamp=results.get("timestamp", ""),
+                    processing_method=results.get("processing_method", "unknown")
+                )
+            else:
+                raise HTTPException(status_code=500, detail=results.get("error", "화자분리 실패"))
+            
+        finally:
+            # 6. 임시 파일 삭제
+            if os.path.exists(audio_path):
+                os.unlink(audio_path)
+                
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)